### DISCUSSION TITLE
light

### DISCUSSION THREAD

**Discussion by AimerZmj** [T-Troll/alienfx-tools#343] [Q&A]
i had set up the light,but the light didnt change,and light effect stays still.Is there any setup i need to change?

> 
![K5DB4%$UTETHY`FJT07J9GA](https://user-images.githubusercontent.com/132764309/236630017-1b23989f-d29e-43ff-923c-6d0dbc7904dd.jpg)
![FYGD{01WXK8E 3D`U}YYZUO_tmb](https://user-images.githubusercontent.com/132764309/236630023-0539ff07-1365-4992-b97c-971f6d71ec96.jpg)

**Comment by T-Troll** [comment]
You have all effects disabled for this profile - see "Software effects" checkbox in the top-right corner.
^_^
Also check effects enabled in Settings.

**Reply by AimerZmj** [reply]
i have click the button but still have no light effect.
![IMG_4103](https://user-images.githubusercontent.com/132764309/236661938-7f8cffcf-0179-4b29-97b1-8e56ba4af59b.JPG)
![IMG_4104](https://user-images.githubusercontent.com/132764309/236661943-75d20855-53f5-40aa-8aef-47621d98db6f.JPG)

**Reply by AimerZmj** [reply]
And light in my keyboard is still after i uninstall the awcc.
![K5DB4%$UTETHY`FJT07J9GA](https://user-images.githubusercontent.com/132764309/236662023-e7719aa7-6437-402d-bd85-63c7d9ac3da4.jpg)

**Comment by T-Troll** [comment]
Ok.
Please open CMD, navigate to app folder and run `alienfx-config backup`. I will provide 4 .reg files with configuration. Please share me `afx-lights.reg` and `afx-gui.reg`.

**Comment by AimerZmj** [comment]
Is the app i have highlighted?
![IMG_4106](https://user-images.githubusercontent.com/132764309/236668889-f5c30eb6-1130-4d4f-8c15-20bfbd9dc4a5.JPG)

**Reply by T-Troll** [reply]
Yes, but you need to open it from CMD (command line).

**Reply by AimerZmj** [reply]
i already run it from CMD,but the software window pops up and disappears quickly,i dont know why.

**Reply by T-Troll** [reply]
First open CMD, then type into it, not just run it from explorer!
Did you even meet the CLI (Command line interface) utilities?

**Reply by AimerZmj** [reply]
sorry for asking so many questions,i m a rookie for computer.

**Reply by T-Troll** [reply]
It's OK. Learn, it's for good! ^_^

**Reply by AimerZmj** [reply]
The system notify like cant find the app.

**Reply by AimerZmj** [reply]
thanks for your patient.

**Reply by AimerZmj** [reply]
is there anything i didnt install for the sitution?

**Reply by T-Troll** [reply]
Read about how to use command line.

**Comment by T-Troll** [comment]
Looking at your latest screenshot, i see now effect enabled (grid color confirm it), but the question is in settings.
First, it's strongly NOT RECOMMENDED to touch "Global speed" slider without serious reason - you will have high CPU load and side effects (you have it already). Set it back to default "100".
Second, i see your zones overlapped. It's ok in general, but each phase only one effect will be set for light - last in query.
Try to turn off effect for overlapped zones temporary.

Hmm... Also... Can you set zone to simple color, not effects? I see you set up grid manually, here can be an issue as well.

PS: BTW, you "Fn" zone have all keyboard lights EXCEPT Fn.
